gensim/models/word2vec.py

```diff
@@ -1216,12 +1216,20 @@ def prepare_vocab(self, hs, negative, wv, update=False, keep_raw_vocab=False, tr
             sorted_vocab = sorted(sorted_vocab_list, key=lambda word: word[1], reverse=True)
```

Comment:
I would still suggest the sorted list be word (keys) only, with counts retrieved via dict-lookups. Constantly accessing the 2nd-item-of-a-tuple, via `[1]`, is less clear about intent.